kpi_scan/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -eu
+
+echo "Finding latest Bearer version"
+curl -H "Accept: application/vnd.github.v3+json" \
+     https://api.github.com/repos/bearer/bearer/releases/latest > /tmp/bearer-release.json
```

Comment:
Wonder if we want to run against canary release too?

Reply:
Maybe. I'll add it to the issue and leave it open.